superset/migrations/shared/migrate_viz/processors.py

```diff
@@ -184,3 +184,32 @@ def _pre_action(self) -> None:
         )
 
         self.data["opacity"] = 0.7
+
+
+class MigrateBubbleChart(MigrateViz):
+    source_viz_type = "bubble"
+    target_viz_type = "bubble_v2"
+    rename_keys = {
+        "bottom_margin": "x_axis_title_margin",
```

Comment:
Unrelated, but what's the reason we have a mix of snake- and camel-case in the chart properties?

Reply:
😏